AllReadyApp/Web-App/AllReady/Services/AttachmentService.cs

```diff
@@ -7,13 +7,16 @@
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Blob;
 using System.Linq;
+using System.Collections.Generic;
 
 namespace AllReady.Services
 {
     public class AttachmentService : IAttachmentService
     {
         private const string ContainerName = "attachments";
-        public static string[] AllowedExtensions = new string[] { ".jpg", ".gif", ".bmp", ".pdf" };
+        private const int MaxAttachmentBytes = 2 * 1024 * 1024; // 2MB
```

Comment:
The maximum size for Azure seems to be 500 TB (https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits)